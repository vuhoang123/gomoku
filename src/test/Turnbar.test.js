import TurnBar from '../components/turnbar';
import renderer from 'react-test-renderer';

test('Turnbar', () => {

  const tree = renderer.create(
    <TurnBar turn={'X'} winner={'X'} reset={() => { }} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
