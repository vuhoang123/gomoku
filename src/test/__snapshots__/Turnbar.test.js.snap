// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Turnbar 1`] = `
<div
  className="turnbar"
>
  <p>
    Winner: X
  </p>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
